.scalafix.conf

```diff
@@ -0,0 +1,13 @@
+rules = [
```

Comment:
Looks like the newer changes have gone missing

Reply:
I noticed some things in build.sbt didn't make it.  I'm now scared of what else didn't.  I don't know what I've done wrong here.

Reply:
I see now.  I think my local series/0.23 wasn't fully up to date.

Reply:
Yep, that's it.  I should have chased my suspicions harder.

```
Head:     series/0.23 Merge pull request #5506 from http4s/non-trivial-merge-2020-10-27
Merge:    origin/series/0.23 Merge pull request #5529 from http4s/non-trivial-merge-2021-11-02
Push:     origin/series/0.23 Merge pull request #5529 from http4s/non-trivial-merge-2021-11-02
```